src/mfa/interfaces/factor.interface.ts

```diff
@@ -11,7 +11,16 @@ export interface Factor {
   type: FactorType;
   sms?: Sms;
   totp?: Totp;
-  userId?: string;
+}
+
+export interface UserManagementFactor {
```

Comment:
move to `user-management` ?

Reply:
same w/ the deserializer etc

Reply:
whoops actually created the files in wrong folder, fixed now